infrastructure/snyk_api/snyk_api.go

```diff
@@ -141,10 +164,26 @@ func (s *SnykApiClientImpl) doCall(method string,
 	return responseBody, nil
 }
 
+func (s *SnykApiClientImpl) processApiResponse(method string, path string, v interface{}) error {
+	responseBody, err := s.doCall("GET", path, nil)
+	if err != nil {
+		return fmt.Errorf("%s: %v: %v", method, err, responseBody)
+	}
+
+	if err := json.Unmarshal(responseBody, v); err != nil {
+		return fmt.Errorf("%s: couldn't unmarshal: %v", method, err)
+	}
+	return nil
+}
+
 func checkResponseCode(r *http.Response) *SnykApiError {
 	if r.StatusCode >= 200 && r.StatusCode <= 399 {
 		return nil
 	}
 
 	return NewSnykApiError("Unexpected response code: "+r.Status, r.StatusCode)
 }
+
+func (f FeatureFlagType) String() string {
+	return string(f)
+}
```

Comment:
Wondering if this is actually needed, or not.

Reply:
I saw this pattern in the policies repo but can see now that for our case, it's unnecessary.
I'm removing it 🙏